about.css

```diff
@@ -0,0 +1,439 @@
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', Arial, Helvetica, sans-serif;
+  color: #272a31;
+  text-decoration: none;
+}
+
+.desktop {
+  display: none;
+}
+
+.bottom-nav {
+  display: none;
+}
+
+.top-nav {
+  display: none;
+}
+
+header {
+  color: #ec5242;
+}
+
+header .fa-bars {
+  margin: 40px;
+}
+
+.background {
+  background: url(./images/img.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  background-position: center;
+  height: auto;
+}
+
+.invisible {
+  display: none;
+}
+
+.menu {
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 90vh;
+  width: 100%;
+  z-index: 2;
+  background: rgba(85, 85, 87, 0.8);
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+
+.menu-ul {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
```

Comment:
- [OPTIONAL ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉